asv_bench/benchmarks/benchmarks.py

```diff
@@ -22,7 +22,15 @@
 import pandas
 
 from .utils import generate_dataframe, RAND_LOW, RAND_HIGH, random_string
-from modin.config import NPartitions
+
+try:
+    from modin.config import NPartitions
+
+    NPARTITIONS = NPartitions.get()
+except ImportError:
+    from modin.config import CpuCount
+
+    NPARTITIONS = CpuCount.get()
```

Comment:
we also may want to bench the modin versions where `modin.config` feature wasn't introduced yet. (`modin.config` was added by #2189 4 months ago)

should we support that old versions of modin?

Reply:
Added support for old version.

Tested with modin 0.7.4.